content/en/about/faq/metrics-and-logs/telemetry-v1-vs-v2.md

```diff
@@ -41,11 +41,14 @@ v2 which are listed below:
   in Mixer-based telemetry. However, more buckets are available by default
   in in-proxy telemetry for latency metrics at the lower latency levels.
 
-* **No metric expiration for short-lived metrics**
+* ~~**No metric expiration for short-lived metrics**~~
```

Comment:
We do not do strikethroughs. If you're striking through, typically means it no longer applies and should be removed.